trigger_webapp/trigger_app/models.py

```diff
@@ -15,16 +50,45 @@ class TriggerEvent(models.Model):
         (E, 'Error'),
         (T, 'Triggered'),
     )
```

Comment:
Why is this defined within the TriggerEvent class, but the similar logic for the source type is in the global scope? Is there a reason for the difference?

Reply:
Similarly with the Project decision model below

Reply:
I put it in the global scope because I want both the VOEvent and TriggerEvent to use the same choices of sources. TriggerEvent no longer needs CHOICES so I deleted it. Only ProjectDecision needs CHOICES so I left it in local scope, I assume this is correct now?